packages/terra-form/docs/Fieldset.md

```diff
@@ -0,0 +1,23 @@
+# Fieldset
+
+Generic form fieldset component which handles the layout of a standard form fieldset including errors, help text, label, value and widget placement.
+
+## Usage
+
+```jsx
+import React from 'react';
+import Fieldset from 'terra-form/lib/Fieldset';
+
+<Fieldset
+  type="checkbox"
```

Comment:
This is outdated. Fixed here. https://github.com/cerner/terra-core/pull/352/commits/33ffe7b0aba15bcf8a600b929fea2d7f7d0c3e30